### ISSUE TITLE
Non Admin users can't set bookmark

### ISSUE THREAD

**Issue by mLorin** [mLorin/LearningProgress#15]
non admin users on e-script cant set bookmark

**Comment by sarahfrederickx** [comment]
Ich habe auf dem Live (https://eskript.ethz.ch/videosinteaching/) und auf mskript (https://mskript.ethz.ch/bonemultiscale/) gerade mit einem Subscriber (sfrederickx@gmx.net) ein Bookmark gesetzt, hat funktioniert. (Das ist ein lokaler User aber es sollte keinen Unterschied für WP machen.)

**Comment by sarahfrederickx** [comment]
Auch mit Shibboleth User funktioniert es einwandfrei.

**Comment by mLorin** [comment]
ich habe es mit meinem normalen account auf escript probiert und es hat nicht funktioniert. Jedoch bei einem anderen buch

**Comment by sarahfrederickx** [comment]
Und beim gleichen Buch?
Gedanken:
A) gewisse Bücher tun blöd oder
B) erst wenn man selber ein Buch eröffnet hätte, würde es funktionieren (einziger Unterschied zwischen deinem normalen User und meinem)

**Comment by sarahfrederickx** [comment]
Was auch noch sein kann, ist, dass du zwar eingeloggt warst, aber auf einem öffentlichen Buch und demnach nicht unbedingt Subscriber in dem Buch warst.

**Comment by mLorin** [comment]
Das mit dem öffentlichen Buch macht am meissten Sinn.
Jedoch kann ich unmöglich realisieren dass es dann gespeichert wird, da WP den user ev. nicht kennt.
Eine Möglichkeit ist dass sich prog für non loged in users nicht dargestellt wird

**Comment by sarahfrederickx** [comment]
Ja: dass sich prog für non loged in users nicht dargestellt wird
Gute Lösung. Sehr tiefe Prio.

Diese Lösung löst nur 80% des Problems (man kann eingeloggt sein und trotzdem nicht im Buch Subscriber sein) und es ist kein wichtiges Problem: Die eSkript als Vorlesungsunterlagen sind 'geschützt' und man muss über Shibboleth einloggen und ist danach immer Subscriber. Solange es für diese User funktioniert, ist alles gut!
